lib/__testfixtures__/property-access.output.js

```diff
@@ -0,0 +1,5 @@
+import { call as Reflect_call } from "std:global/Reflect";
```

Comment:
Great test! The fact that it doesn't change .length is really important, but I totally forgot that was something we had to worry about. I'm glad that this worked out!

Reply:
Thanks